content/docs/1.1.0/deploy/install/_index.md

```diff
@@ -21,7 +21,7 @@ For information on deploying Longhorn on specific nodes and rejecting general wo
 Each node in the Kubernetes cluster where Longhorn is installed must fulfill the following requirements:
 
 -  A container runtime compatible with Kubernetes (Docker v1.13+, containerd v1.3.7+, etc.)
--  Kubernetes v1.16+.
+-  Kubernetes v1.16+ (only in control plane node).
     - Recommend Kubernetes v1.17+
```

Comment:
Ideally yes, but I am not certain we can claim this way because of our regression all relying on the same version base including kubelet. 

So it's better just follow up https://kubernetes.io/releases/version-skew-policy/, but here we just claim the min version required.